### ISSUE TITLE
Function execution logs

### ISSUE THREAD

**Issue by worldspawn** [Azure/azure-functions-docker#134]
I have an sub-orchestration calling a sub-orchestration that calls another orchestration over http. At the top sub orchestration I am getting an error:

```
Microsoft.Azure.WebJobs.Host.FunctionInvocationException: Exception while executing function: PdfTextOrchestration ---> DurableTask.Core.Exceptions.OrchestrationFailureException: Orchestrator function 'PdfTextOrchestration' failed: The orchestrator function 'RunRemoteOrchestration' failed: "". See the function execution logs for additional details.
```

Despite covering the entirety of RunRemoteOrchestration in a try catch that logs any exception I am never getting any errors logged from it. I would live to know where additional details are stored `See the function execution logs for additional details.` 

I have seen some guidance that says there should be a file share in the storage account but I dont see any file shares.

**Comment by worldspawn** [comment]
Heres RunRemoteOrchestration. I *never* get a log entry from the catch in there.

```
[FunctionName("RunRemoteOrchestration")]
        public async Task<JToken> RunRemoteOrchestration([OrchestrationTrigger] DurableOrchestrationContext context)
        {
            try {
                var input = context.GetInput<JObject>();
                string statusQueryGetUri = null;
                _logger.LogTrace("Executing RunRemoteOrchestration");

                if (input.SelectToken("statusQueryGetUri") == null) {
                    _logger.LogTrace("Executing RunRemoteOrchestration - calling Start");
                    statusQueryGetUri = await context.CallActivityAsync<string>("StartRemoteOrchestration", input);
                    input.Add("statusQueryGetUri", new JValue(statusQueryGetUri));
                }
                else {
                    _logger.LogTrace("Executing RunRemoteOrchestration - statusUriExists {input}", input);
                    statusQueryGetUri = input.SelectToken("statusQueryGetUri").Value<string>();
                }

                var result = await  context.CallActivityAsync<(bool success, string output)>("CheckIsComplete", statusQueryGetUri);
                if (result.success) {                
                    return JToken.Parse(result.output);                
                }

                context.ContinueAsNew(input);
                return null;
            }
            catch (Exception ex) {
                _logger.LogError(ex, "RunRemoteOrchestration Failed.");
                throw;
            }
        }
```

Also theres no errors reported from the function it is supposed to be calling.

**Comment by worldspawn** [comment]
So, aside my logging difficulties I figured out what my issue was. I have the function instance this function instance calls into behind a kubernetes service (a load balancer) so the the returned code wasn't necessarily the correct code on the instance it would hit on next cycle.

I ran through the guidance here: https://github.com/Azure/azure-functions-host/issues/4147#issuecomment-477431016

Option 2 then applied the kubernetes flavor to it and it is now working. All my func calls are going through and it would seem all is well...

~~...except when I grab a status uri eg. `http://dev-source-file-processing-pdf/runtime/webhooks/durabletask/instances/7faa08f05f2f4261ba536d937c9e4c4c?taskHub=PdfProcessingHub&connection=Storage&code=6821ad712805468ca7bb0e51e3ef5a26` and manually curl it I always get 401.

Everything seems to be working in but the fact that my manual check fails is perplexing/worrying. Probably outside the scope of this repo though.~~

In bash land ampersands need to be wrapped in quotes.
